app/services/reporting/total_user_count_report.rb

```diff
@@ -23,10 +30,22 @@ def total_user_count_emailable_report
       )
     end
 
+    def total_verified_users_emailable_report
+      EmailableReport.new(
+        email_options: { title: 'Total verified users' },
+        table: total_verified_users_report,
+        csv_name: 'total_verified_users',
+      )
+    end
+
     private
 
     def total_user_count
       User.where('created_at <= ?', report_date).count
     end
+
+    def verified_user_count
+      Profile.where(active: true).where('activated_at <= ?', report_date).count
+    end
```

Comment:
I am boldly just assuming that the request for "verified users" means "IdV users," but hopefully @zachmargolis or someone can confirm if that is how others would interpret the term?

And to slightly further open the can of worms: I'm including the `active: true` limit [to be consistent with the handbook](https://handbook.login.gov/articles/queries.html#idv-users), but the other report generated in this class is "total users all time" so I think one could make an argument that we shouldn't filter only on `active: true` profiles.

Reply:
yeah that's a correct interpretation and yes, we should keep the handbook query here